pkg/operator/ceph/cluster/controller.go

```diff
@@ -527,23 +528,42 @@ func (c *ClusterController) checkPVPresentInCluster(drivers []string, clusterID
 	return false, nil
 }
 
-// removeFinalizer removes a finalizer
-func removeFinalizer(client client.Client, name types.NamespacedName) error {
-	cephCluster := &cephv1.CephCluster{}
-	err := client.Get(context.TODO(), name, cephCluster)
+func removeFinalizers(client client.Client, name types.NamespacedName) error {
+	// Remove cephcluster finalizer
+	err := removeFinalizer(client, name, &cephv1.CephCluster{})
+	if err != nil {
+		return errors.Wrap(err, "failed to remove cephcluster finalizer")
+	}
+
+	// Remove monSecretsFinalizer
+	name = types.NamespacedName{Name: mon.AppName, Namespace: name.Namespace}
+	err = removeFinalizer(client, name, &corev1.Secret{})
+	if err != nil {
+		return errors.Wrap(err, "failed to remove mon secret finalizer")
+	}
+
+	// Remove monConfigMapFinalizer
+	name = types.NamespacedName{Name: mon.EndpointConfigMapName, Namespace: name.Namespace}
+	err = removeFinalizer(client, name, &corev1.ConfigMap{})
+	if err != nil {
+		return errors.Wrap(err, "failed to remove mon configmap finalizer")
```

Comment:
add resource name to the log as well -  `failed to remove finalizer for the configmap <configmap name>`

Reply:
Sure, added.